src/uint/neg.rs

```diff
@@ -12,7 +17,7 @@ impl<const LIMBS: usize> Uint<LIMBS> {
             carry = r >> Limb::BITS;
             i += 1;
         }
-        Uint::new(ret)
+        (Uint::new(ret), carry as Word)
```

Comment:
hmmm, interesting :thinking:

Reply:
The other solution is to name whatever overflows as not carry nor borrow, because these are well defined; but that would be somewhat not standard abuse :)

Reply:
Done!